regression/Makefile

```diff
@@ -1,10 +1,12 @@
 
 DIRS = ansi-c \
        cbmc \
+      cbmc-typedef \
        cpp \
        cbmc-java \
        goto-analyzer \
        goto-instrument \
+       goto-instrument-typedef \
```

Comment:
Do we actually need to have these in new subfolders? Wouldn't the cbmc and goto-instrument folders do?